e2e/example/next.config.js

```diff
@@ -0,0 +1,2 @@
+const withCSS = require('@zeit/next-css')
```

Comment:
I think you can remove this file. In the new version of next.js you can use CSS without the need of any plugin. Moreover, you are not using any CSS file for the example, aren't you?